### PR TITLE
feat: add feedback messages on login

### PR DESCRIPTION
**Pull request by eordano**
https://explorer.decentraland.zone/branch/messages/index.html

### CODE REVIEW

File: packages/shared/passports/transformations/processServerProfile.ts [hunk 1]
```diff
@@ -54,10 +54,7 @@ export function processServerProfile(userId: string, receivedProfile: any): Prof
       hairColor: colorString(receivedProfile.avatar.hair.color),
       skinColor: colorString(receivedProfile.avatar.skin.color),
       bodyShape: fixWearableIds(receivedProfile.avatar.bodyShape),
-      wearables: receivedProfile.avatar.wearables
-        .map(fixWearableIds)
-        .filter(dropDeprecatedWearables)
-        .filter(noExclusiveMismatches(receivedProfile.inventory || []))
+      wearables: receivedProfile.avatar.wearables.map(fixWearableIds).filter(dropDeprecatedWearables)
```
Comment (eordano) [inline packages/shared/passports/transformations/processServerProfile.ts]: Oops...

File: packages/shared/loading/actions.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { action } from 'typesafe-actions'
+
+export const SCENE_LOAD = 'Loading scene'
+export const SCENE_START = 'Started scene'
+export const SCENE_FAIL = 'Loading scene'
```
Comment (moliva) [inline packages/shared/loading/actions.ts]:
```suggestion
export const SCENE_FAIL = 'Failed scene'
```

File: packages/shared/passports/transformations/processServerProfile.ts [hunk 3]
```diff
@@ -54,10 +58,7 @@ export function processServerProfile(userId: string, receivedProfile: any): Prof
       hairColor: colorString(receivedProfile.avatar.hair.color),
       skinColor: colorString(receivedProfile.avatar.skin.color),
       bodyShape: fixWearableIds(receivedProfile.avatar.bodyShape),
-      wearables: receivedProfile.avatar.wearables
-        .map(fixWearableIds)
-        .filter(dropDeprecatedWearables)
-        .filter(noExclusiveMismatches(receivedProfile.inventory || []))
+      wearables: wearables
```
Comment (moliva) [inline packages/shared/passports/transformations/processServerProfile.ts]:
```suggestion
      wearables
```

File: packages/shared/world/parcelSceneManager.ts [hunk 4]
```diff
@@ -83,7 +82,7 @@ export async function enableParcelSceneLoading(options: EnableParcelSceneLoading
   ret.on('Scene.shouldStart', async (opts: { sceneId: string }) => {
     const sceneId = opts.sceneId
     const parcelSceneToStart = await ret.getParcelData(sceneId)
-
+    ;(global as any)['globalStore'].dispatch({ type: 'Loading scene', sceneId })
```
Comment (moliva) [inline packages/shared/world/parcelSceneManager.ts]: Use helper functions here?
```suggestion
    globalSignalSceneLoad(sceneId)
```

File: packages/shared/world/parcelSceneManager.ts [hunk 5]
```diff
@@ -97,6 +96,7 @@ export async function enableParcelSceneLoading(options: EnableParcelSceneLoading
         sceneLifeCycleObservable.remove(observer)
         clearForegroundTimeout(timer)
         ret.notify('Scene.status', sceneStatus)
+        ;(global as any)['globalStore'].dispatch({ type: 'Started scene', sceneId })
```
Comment (moliva) [inline packages/shared/world/parcelSceneManager.ts]:
```suggestion
        globalSignalSceneStart(sceneId)
```

File: packages/shared/world/parcelSceneManager.ts [hunk 6]
```diff
@@ -109,6 +109,7 @@ export async function enableParcelSceneLoading(options: EnableParcelSceneLoading
       const worker = getSceneWorkerBySceneID(sceneId)
       if (worker && !worker.sceneStarted) {
         sceneLifeCycleObservable.remove(observer)
+        ;(global as any)['globalStore'].dispatch({ type: 'Failed scene', sceneId })
```
Comment (moliva) [inline packages/shared/world/parcelSceneManager.ts]:
```suggestion
        globalSignalSceneFail(sceneId)
```

**Review by moliva — APPROVED** [review]

**Pull request merged**